src/bin/lib/subcommands.c/commandline.c

```diff
@@ -61,46 +68,47 @@ commandline_run(CommandLine *command, int argc, char **argv)
 
 	if (command->run != NULL)
 	{
-		return command->run(argc, argv);
+		command->run(argc, argv);
+		return true;
 	}
-	else if (argc == 0)
+
+	if (argc == 0)
 	{
 		/*
 		 * We're at the end of the command line already, and command->run is
 		 * not set, which means we expected a subcommand to be used, but none
 		 * have been given by the user. Inform him.
 		 */
-		commandline_print_subcommands(command, stderr);
+		fprintf(stderr, "%s: expected a command\n", breadcrumb);
```

Comment:
Should we use `arg0` in this error message, maybe?

Reply:
The output for this command:
```
pg_autoctl create
```
Looks like this:
```
pg_autoctl create: expected a command

Available commands:
  pg_autoctl create
    monitor    Initialize a pg_auto_failover monitor node
    postgres   Initialize a pg_auto_failover standalone postgres node
    formation  Create a new formation on the pg_auto_failover monitor
```

And for this command:
```
pg_autoctl
```
it looks like this:
```
pg_autoctl: expected a command

Available commands:
  pg_autoctl
  + create   Create a pg_auto_failover node, or formation
  + drop     Drop a pg_auto_failover node, or formation
  + config   Manages the pg_autoctl configuration
  + show     Show pg_auto_failover information
  + enable   Enable a feature on a formation
  + disable  Disable a feature on a formation
  + perform  Perform an action orchestrated by the monitor
    run      Run the pg_autoctl service (monitor or keeper)
    stop     signal the pg_autoctl service for it to stop
    reload   signal the pg_autoctl for it to reload its configuration
    help     print help message
    version  print pg_autoctl version
```

Again, if you have a command that seems to have a wrong error, just let me know.